pyproject.toml

```diff
@@ -268,6 +269,9 @@ ignore = [
     "UP024", # Replace aliased errors with `OSError`
 ]
 
+[tool.ruff.flake8-pytest-style]
+fixture-parentheses = false
```

Comment:
This makes it inconsistent with argument fixtures. Is there an advantage of this over the default?
`@pytest.fixture(name=XXX)`
`@pytest.fixture(autouse=True)`
versus
`@pytest.fixture`

Reply:
> I think the PT001 rule is a good one, but I'm not sure about overriding the default style.

I've polled Martin, Paulus and they both preferred the non-parentheses style (so do I). Additionally, it seems everybody prefers that @ HA, as there are just a couple that have them (and thus adjusted in this PR). Almost all fixture don't have parentheses.

Reply:
Cool 🙇